src/py/migraphx_py.cpp

```diff
@@ -214,7 +214,7 @@ MIGRAPHX_PYBIND11_MODULE(migraphx, m)
         .def(py::init([](py::kwargs kwargs) {
             auto v    = migraphx::to_value(kwargs);
             auto t    = migraphx::shape::parse_type(v.get("type", std::string{"float"}));
-            auto lens = v.get<std::size_t>("lens", {1});
+            auto lens = v.get<std::size_t>("lens", std::vector<size_t>{1});
```

Comment:
I think with #1141 this is not necessary.

Reply:
Yes, I rebased with respect to `value_string_literal`.  PR should be merged after #1141.